### ISSUE TITLE
Callback URL for Single Page Application

### ISSUE THREAD

**Issue by LetsDoIt-** [sahat/satellizer#652]
Hi I am working on application with Firebase as the server and don't have a dedicated server assigned.
I created a light server but i don't know how to handle the callback url for linkedin to redirect the user to the previous page.

**Comment by sahat** [comment]
I am not familiar with how Firebase works or how their "light servers" work, but I am pretty sure this won't work unless you implement authentication on the back-end. If you look at the examples in the repository, you will see that there are server-side examples with routes like `/auth/facebook`, `/auth/linkedin`, etc. Satellizer (on the client-side) just opens the pop-up window and requests `code` from a OAuth provider, but it's up the back-end to exchange that code for access_token and use that access_token to fetch user information.

Long story short, you will need a back-end service with those routes implemented. Again, see the `examples`  directory.
